bsonnumpy/bson/bson-config.h

```diff
@@ -92,7 +110,10 @@
 /*
  * Define to 1 if you have snprintf available on your platform.
  */
-#define BSON_HAVE_SNPRINTF 1
+#ifndef MS_WINDOWS
+# define BSON_HAVE_SNPRINTF 1
```

Comment:
Does this need an if/else so that BSON_HAVE_SNPRINTF is defined to 0 on windows?

Reply:
Done.

Reply:
I meant:
```
#ifdef MS_WINDOWS
# define BSON_HAVE_SNPRINTF 0
#else
# define BSON_HAVE_SNPRINTF 1
#endif
```

Reply:
Fixed.